### PR TITLE
Remove build file generation

### PR DESCRIPTION
**Pull request by GMNGeoffrey**
With https://github.com/google/llvm-bazel/pull/11, we will use
handwritten Bazel BUILD files.

### CODE REVIEW

**Review by chandlerc — APPROVED** [review]
LGTM (once #11 lands of course)

**Comment by GMNGeoffrey** [comment]
I actually want to merge this one before #11 because I currently have things set up to traverse LLVM commits and run the generation script, tagging new commits on this repo. Don't want that to happen when #11 is merged. It will append the generated build file to your handwritten one

**Pull request merged**
